Sources/Orbit/Components/Slider.swift

```diff
@@ -0,0 +1,421 @@
+    }
+
+    @ViewBuilder func secondThumb(atPosition horisontalPosition: CGFloat, sliderWidth: CGFloat) -> some View {
+        thumb(
+            horisontalPosition: horisontalPosition,
+            sliderWidth: sliderWidth,
+            isDragging: $isSecondThumbDrag
+        ) {
+            self.secondValue = $0
+        }
+        .zIndex(isSecondThumbDrag ? 1 : 0)
+
+    }
+
+    @ViewBuilder func thumb(
+        horisontalPosition: CGFloat,
+        sliderWidth: CGFloat,
+        isDragging: Binding<Bool>,
+        onValueChange: @escaping (Value) -> Void
+    ) -> some View {
+        SliderThumb()
+            .position(x: horisontalPosition, y: lineCenter)
+            .highPriorityGesture(
+                DragGesture(minimumDistance: 1)
+                    .onChanged { value in
+
+                        if isDragging.wrappedValue == false {
+                            dragStartOffset = value.startLocation.x - horisontalPosition
+                            isDragging.wrappedValue = true
+                        }
+
+                        let newHorisontalPosition = value.location.x - dragStartOffset
+
+                        guard newHorisontalPosition < sliderWidth else {
+                            onValueChange(bounds.upperBound)
+                            return
+                        }
+
+                        guard newHorisontalPosition > 0 else {
+                            onValueChange(bounds.lowerBound)
+                            return
+                        }
+
+                        if step != nil {
+                            let currentValue = currentValue(location: newHorisontalPosition, sliderWidth: sliderWidth)
+                            guard let bucket = buckets.first(where: { $0.contains(currentValue) }) else { return  }
+
+                            onValueChange(currentValue < bucket.middle ? bucket.lowerBound : bucket.upperBound)
+                        } else {
+                            onValueChange(currentValue(location: newHorisontalPosition, sliderWidth: sliderWidth))
+                        }
+                    }
+                    .onEnded { value in
+                        isDragging.wrappedValue = false
+                        dragStartOffset = 0
+                    }
+        )
+    }
+
+    @ViewBuilder func strokedPath(from: CGFloat, to: CGFloat) -> some View {
+        Path { path in
+            path.move(to: .init(x: from, y: lineCenter))
+            path.addLine(to: .init(x: to, y: lineCenter))
+        }
+        .stroke(Color.blueNormal, lineWidth: lineHeight)
+    }
+
+    private var lineHeight: CGFloat {
+        CGFloat.xxSmall * sizeCategory.controlRatio
+    }
+
+    private var lineCenter: CGFloat {
+        lineHeight / 2
+    }
+
+    private var thumbOverlapping: CGFloat {
+        (SliderThumb.circleDiameter / 2 - lineCenter) * sizeCategory.controlRatio
+    }
+
+    private var accessibilityHint: String {
+        startLabel + "—" + endLabel
```

Comment:
Don't we need some spaces around the dash?

Also I'd probably use string interpolation.